src/interface/src/app/account/change-password/change-password.component.html

```diff
@@ -0,0 +1,76 @@
+<h4 class="form-field-title">Password</h4>
+<form
+  [formGroup]="form"
+  (ngSubmit)="saveForm()"
+  *ngIf="state === 'editing' || state === 'saving'">
+  <div class="form-field-label">
+    Current Password
+    <div class="required-blurb">required</div>
+  </div>
+  <mat-form-field appearance="outline" class="form-field-input">
+    <input type="text" required formControlName="current" matInput />
+  </mat-form-field>
+  <div class="form-field-label">
+    New Password
+    <div class="required-blurb">required</div>
+  </div>
+  <mat-form-field appearance="outline" class="form-field-input">
+    <input type="text" required formControlName="password" matInput />
+  </mat-form-field>
+  <div class="form-field-label">
+    Confirm Password
+    <div class="required-blurb">required</div>
+  </div>
+  <mat-form-field appearance="outline" class="form-field-input">
+    <input type="text" required formControlName="passwordConfirm" matInput />
+  </mat-form-field>
+  <div>
+    <button mat-flat-button color="primary">SAVE</button>
+    <button
+      type="button"
+      mat-stroked-button
+      color="secondary"
+      (click)="state = 'view'"
+      class="btn-spaced">
+      Cancel
+    </button>
+  </div>
+
+  <!-- Unified error output -->
+  <mat-error *ngIf="form.errors || error">
```

Comment:
Since our UX designs put this user-facing text within a little tooltip/popup (I can't remember what we're calling it :sweat_smile: ), I've been wondering how we should approach writing the collection of user-facing info -- maybe passing a vector of these messages or maybe ng-content, or something else?

I personally love the UI that shows which of these requirements are satisfied and which have errors, maybe with checkbox icons and Xs and colors, but that's not in our spec.

Reply:
ah sorry i didnt clarify this piece is a placeholder (just debugging to check I was getting the right errors).
Ultimately this needs that little tooltip/popup thing.

<img width="496" alt="Screen Shot 2023-12-06 at 13 12 05" src="https://github.com/OurPlanscape/Planscape/assets/358892/bf310393-79ab-4881-8b3d-90155d8cc48a">

>  I've been wondering how we should approach writing the collection of user-facing info -- maybe passing a vector of these messages or maybe ng-content, or something else?

Good question, by looking at the examples on figma I dont see a scenario in which we show multiple lines of errors on this component, but maybe we can future proof it a bit with having an array of messages as input